### DISCUSSION TITLE
registry/lib/components/video/player/common/speed/context.ts如何开发

### DISCUSSION THREAD

**Discussion by imshixin** [the1812/Bilibili-Evolved#3321] [综合讨论 / General]
https://github.com/the1812/Bilibili-Evolved/blob/0c128ba3f2d1a51eb1c20d5e630e9ca24644bbb2/registry/lib/components/video/player/common/speed/context.ts

这个文件应该运行哪个任务来更新呢

watch-feature貌似找不到这个文件

**Comment by the1812** [comment]
这不是组件, 只是个公共模块

**Reply by imshixin** [reply]
所以怎么更新它呢

**Reply by the1812** [reply]
看哪个组件引用到了, 就启动那个组件, 这个应该就是倍速相关的那两个用到
